next-app/src/app/edit/page.tsx

```diff
@@ -0,0 +1,253 @@
+import Image from 'next/image'
+import IconHash from '@assets/icons/hash.svg'
+import IconB from '@assets/icons/bold.svg'
+import IconI from '@assets/icons/italic.svg'
+import IconU from '@assets/icons/underline.svg'
+import IconS from '@assets/icons/strikethrough.svg'
+import IconColor from '@assets/icons/color.svg'
+import IconImage from '@assets/icons/image.svg'
+import IconLink from '@assets/icons/link.svg'
+import IconList from '@assets/icons/list.svg'
+import IconNumList from '@assets/icons/num_list.svg'
+import IconInline from '@assets/icons/inline.svg'
+import IconBlockcode from '@assets/icons/blockcode.svg'
+
+import IconBurger from '@assets/icons/burger.svg'
+import IconSetting from '@assets/icons/settings.svg'
+import IconLogout from '@assets/icons/logout.svg'
+
+import IconTitle from '@assets/icons/title.svg'
+import IconReportNum from '@assets/icons/report_num.svg'
+import IconPageNum from '@assets/icons/page_num.svg'
+import IconDate from '@assets/icons/date.svg'
+
+import IconPDF from '@assets/icons/pdf.svg'
+import IconArrow from '@assets/icons/arrow.svg'
+
+export default function Edit() {
+  return (
+    <main className='w-full h-screen bg-white text-black flex flex-col items-center'>
+      {/* ヘッダー */}
+      <div className='w-full h-16 bg-gray-200 flex justify-between items-center'>
+        <div className='mx-6'>
+          <h1 className='text-2xl'>まーくだうんぜみれぽ</h1>
+        </div>
+        <div className='mx-6 flex gap-5'>
+          <Image
+            src={IconBurger}
+            alt='burger'
+          />
+          <Image
+            src={IconSetting}
+            alt='setting'
+          />
+          <Image
+            src={IconLogout}
+            alt='logout'
+          />
+        </div>
+      </div>
+
+      {/* ページの本体部分 */}
+      <div className='w-full h-full p-6'>
+        <div className='w-full h-full grid grid-cols-2 justify-between gap-0 border-2 border-gray-300 rounded-lg overflow-hidden'>
+          {/* エディター */}
+          <div>
```

Comment:
[必須]

145行目の対応のために `flex` をつける
```suggestion
          <div className='flex'>
```

Reply:
5ace172d18e84eb06b13a5d68c8cd37c3b4e8789 で修正しました👍